test/test-filter-process.sh

```diff
@@ -0,0 +1,75 @@
+#!/usr/bin/env bash
+
+. "test/testlib.sh"
+
+if [ "$GIT_LFS_USE_LEGACY_FILTER" == "1" ]; then
+  echo "skip: $0 (filter process disabled)"
+  exit
+fi
```

Comment:
Ok. Please note that all other tests should already use the filter protocol in all tests of this branch because the protocol config is set and takes precedence if supported by Git